Dockerfile

```diff
@@ -10,7 +10,18 @@ USER root
 RUN apt-get install -y julia libnettle4 && apt-get clean
 
 # R dependencies that conda can't provide (X, fonts)
-RUN apt-get install -y libxrender1 fonts-dejavu && apt-get clean
+RUN apt-get install -y --no-install-recommends libxrender1 fonts-dejavu && apt-get clean
```

Comment:
Running the examples in the R example notebook did results in messed up fonts. I'll add that option back and then check the results.